examples/bin/verify-java

```diff
@@ -28,10 +28,10 @@ sub fail_check {
     : "Make sure that \"java\" is installed and on your PATH.";
 
   print STDERR <<"EOT";
-Druid requires Java 8. $current_version_text
+Druid only officially supports Java 8. Any Java version later than 8 is still experimental. $current_version_text
```

Comment:
May be good to make a similar update to the docs (e.g., https://github.com/apache/druid/blob/master/docs/tutorials/index.md#software)

Reply:
Done